docs/commands-details/workload_create_update_apply.md

```diff
@@ -88,7 +88,7 @@ Update workload:
 18, 17   |        tag: tap-1.1
 ...
 
-? Really update the workload "spring-pet-clinic"? (y/N)
+? Really update the workload "spring-pet-clinic"? [yN]
```

Comment:
I would be inclined to ask design if it's correct to use the survey this way (`[yN]`)

Reply:
@danfein please a check here